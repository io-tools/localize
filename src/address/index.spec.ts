import { getCountries } from "./index";

describe("Address localization", () => {
  it("should return countries supported", () => {
    const countries = getCountries();
    expect(countries).toMatchSnapshot();
  });
});
