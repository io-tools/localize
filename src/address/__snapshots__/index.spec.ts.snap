// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Address localization should return countries supported 1`] = `
Array [
  Object {
    "code": "AC",
    "name": "ASCENSION ISLAND",
  },
  Object {
    "code": "AD",
    "name": "ANDORRA",
  },
  Object {
    "code": "AE",
    "name": "UNITED ARAB EMIRATES",
  },
  Object {
    "code": "AF",
    "name": "AFGHANISTAN",
  },
  Object {
    "code": "AG",
    "name": "ANTIGUA AND BARBUDA",
  },
  Object {
    "code": "AI",
    "name": "ANGUILLA",
  },
  Object {
    "code": "AL",
    "name": "ALBANIA",
  },
  Object {
    "code": "AM",
    "name": "ARMENIA",
  },
  Object {
    "code": "AO",
    "name": "ANGOLA",
  },
  Object {
    "code": "AQ",
    "name": "ANTARCTICA",
  },
  Object {
    "code": "AR",
    "name": "ARGENTINA",
  },
  Object {
    "code": "AS",
    "name": "AMERICAN SAMOA",
  },
  Object {
    "code": "AT",
    "name": "AUSTRIA",
  },
  Object {
    "code": "AU",
    "name": "AUSTRALIA",
  },
  Object {
    "code": "AW",
    "name": "ARUBA",
  },
  Object {
    "code": "AX",
    "name": "FINLAND",
  },
  Object {
    "code": "AZ",
    "name": "AZERBAIJAN",
  },
  Object {
    "code": "BA",
    "name": "BOSNIA AND HERZEGOVINA",
  },
  Object {
    "code": "BB",
    "name": "BARBADOS",
  },
  Object {
    "code": "BD",
    "name": "BANGLADESH",
  },
  Object {
    "code": "BE",
    "name": "BELGIUM",
  },
  Object {
    "code": "BF",
    "name": "BURKINA FASO",
  },
  Object {
    "code": "BG",
    "name": "BULGARIA (REP.)",
  },
  Object {
    "code": "BH",
    "name": "BAHRAIN",
  },
  Object {
    "code": "BI",
    "name": "BURUNDI",
  },
  Object {
    "code": "BJ",
    "name": "BENIN",
  },
  Object {
    "code": "BL",
    "name": "SAINT BARTHELEMY",
  },
  Object {
    "code": "BM",
    "name": "BERMUDA",
  },
  Object {
    "code": "BN",
    "name": "BRUNEI DARUSSALAM",
  },
  Object {
    "code": "BO",
    "name": "BOLIVIA",
  },
  Object {
    "code": "BQ",
    "name": "BONAIRE, SINT EUSTATIUS, AND SABA",
  },
  Object {
    "code": "BR",
    "name": "BRAZIL",
  },
  Object {
    "code": "BS",
    "name": "BAHAMAS",
  },
  Object {
    "code": "BT",
    "name": "BHUTAN",
  },
  Object {
    "code": "BV",
    "name": "BOUVET ISLAND",
  },
  Object {
    "code": "BW",
    "name": "BOTSWANA",
  },
  Object {
    "code": "BY",
    "name": "BELARUS",
  },
  Object {
    "code": "BZ",
    "name": "BELIZE",
  },
  Object {
    "code": "CA",
    "name": "CANADA",
  },
  Object {
    "code": "CC",
    "name": "COCOS (KEELING) ISLANDS",
  },
  Object {
    "code": "CD",
    "name": "CONGO (DEM. REP.)",
  },
  Object {
    "code": "CF",
    "name": "CENTRAL AFRICAN REPUBLIC",
  },
  Object {
    "code": "CG",
    "name": "CONGO (REP.)",
  },
  Object {
    "code": "CH",
    "name": "SWITZERLAND",
  },
  Object {
    "code": "CI",
    "name": "COTE D'IVOIRE",
  },
  Object {
    "code": "CK",
    "name": "COOK ISLANDS",
  },
  Object {
    "code": "CL",
    "name": "CHILE",
  },
  Object {
    "code": "CM",
    "name": "CAMEROON",
  },
  Object {
    "code": "CN",
    "name": "CHINA",
  },
  Object {
    "code": "CO",
    "name": "COLOMBIA",
  },
  Object {
    "code": "CR",
    "name": "COSTA RICA",
  },
  Object {
    "code": "CU",
    "name": "CUBA",
  },
  Object {
    "code": "CV",
    "name": "CAPE VERDE",
  },
  Object {
    "code": "CW",
    "name": "CURACAO",
  },
  Object {
    "code": "CX",
    "name": "CHRISTMAS ISLAND",
  },
  Object {
    "code": "CY",
    "name": "CYPRUS",
  },
  Object {
    "code": "CZ",
    "name": "CZECH REP.",
  },
  Object {
    "code": "DE",
    "name": "GERMANY",
  },
  Object {
    "code": "DJ",
    "name": "DJIBOUTI",
  },
  Object {
    "code": "DK",
    "name": "DENMARK",
  },
  Object {
    "code": "DM",
    "name": "DOMINICA",
  },
  Object {
    "code": "DO",
    "name": "DOMINICAN REP.",
  },
  Object {
    "code": "DZ",
    "name": "ALGERIA",
  },
  Object {
    "code": "EC",
    "name": "ECUADOR",
  },
  Object {
    "code": "EE",
    "name": "ESTONIA",
  },
  Object {
    "code": "EG",
    "name": "EGYPT",
  },
  Object {
    "code": "EH",
    "name": "WESTERN SAHARA",
  },
  Object {
    "code": "ER",
    "name": "ERITREA",
  },
  Object {
    "code": "ES",
    "name": "SPAIN",
  },
  Object {
    "code": "ET",
    "name": "ETHIOPIA",
  },
  Object {
    "code": "FI",
    "name": "FINLAND",
  },
  Object {
    "code": "FJ",
    "name": "FIJI",
  },
  Object {
    "code": "FK",
    "name": "FALKLAND ISLANDS (MALVINAS)",
  },
  Object {
    "code": "FM",
    "name": "MICRONESIA (Federated State of)",
  },
  Object {
    "code": "FO",
    "name": "FAROE ISLANDS",
  },
  Object {
    "code": "FR",
    "name": "FRANCE",
  },
  Object {
    "code": "GA",
    "name": "GABON",
  },
  Object {
    "code": "GB",
    "name": "UNITED KINGDOM",
  },
  Object {
    "code": "GD",
    "name": "GRENADA (WEST INDIES)",
  },
  Object {
    "code": "GE",
    "name": "GEORGIA",
  },
  Object {
    "code": "GF",
    "name": "FRENCH GUIANA",
  },
  Object {
    "code": "GG",
    "name": "CHANNEL ISLANDS",
  },
  Object {
    "code": "GH",
    "name": "GHANA",
  },
  Object {
    "code": "GI",
    "name": "GIBRALTAR",
  },
  Object {
    "code": "GL",
    "name": "GREENLAND",
  },
  Object {
    "code": "GM",
    "name": "GAMBIA",
  },
  Object {
    "code": "GN",
    "name": "GUINEA",
  },
  Object {
    "code": "GP",
    "name": "GUADELOUPE",
  },
  Object {
    "code": "GQ",
    "name": "EQUATORIAL GUINEA",
  },
  Object {
    "code": "GR",
    "name": "GREECE",
  },
  Object {
    "code": "GS",
    "name": "SOUTH GEORGIA",
  },
  Object {
    "code": "GT",
    "name": "GUATEMALA",
  },
  Object {
    "code": "GU",
    "name": "GUAM",
  },
  Object {
    "code": "GW",
    "name": "GUINEA-BISSAU",
  },
  Object {
    "code": "GY",
    "name": "GUYANA",
  },
  Object {
    "code": "HK",
    "name": "HONG KONG",
  },
  Object {
    "code": "HM",
    "name": "HEARD AND MCDONALD ISLANDS",
  },
  Object {
    "code": "HN",
    "name": "HONDURAS",
  },
  Object {
    "code": "HR",
    "name": "CROATIA",
  },
  Object {
    "code": "HT",
    "name": "HAITI",
  },
  Object {
    "code": "HU",
    "name": "HUNGARY (Rep.)",
  },
  Object {
    "code": "ID",
    "name": "INDONESIA",
  },
  Object {
    "code": "IE",
    "name": "IRELAND",
  },
  Object {
    "code": "IL",
    "name": "ISRAEL",
  },
  Object {
    "code": "IM",
    "name": "ISLE OF MAN",
  },
  Object {
    "code": "IN",
    "name": "INDIA",
  },
  Object {
    "code": "IO",
    "name": "BRITISH INDIAN OCEAN TERRITORY",
  },
  Object {
    "code": "IQ",
    "name": "IRAQ",
  },
  Object {
    "code": "IR",
    "name": "IRAN",
  },
  Object {
    "code": "IS",
    "name": "ICELAND",
  },
  Object {
    "code": "IT",
    "name": "ITALY",
  },
  Object {
    "code": "JE",
    "name": "CHANNEL ISLANDS",
  },
  Object {
    "code": "JM",
    "name": "JAMAICA",
  },
  Object {
    "code": "JO",
    "name": "JORDAN",
  },
  Object {
    "code": "JP",
    "name": "JAPAN",
  },
  Object {
    "code": "KE",
    "name": "KENYA",
  },
  Object {
    "code": "KG",
    "name": "KYRGYZSTAN",
  },
  Object {
    "code": "KH",
    "name": "CAMBODIA",
  },
  Object {
    "code": "KI",
    "name": "KIRIBATI",
  },
  Object {
    "code": "KM",
    "name": "COMOROS",
  },
  Object {
    "code": "KN",
    "name": "SAINT KITTS AND NEVIS",
  },
  Object {
    "code": "KP",
    "name": "NORTH KOREA",
  },
  Object {
    "code": "KR",
    "name": "SOUTH KOREA",
  },
  Object {
    "code": "KW",
    "name": "KUWAIT",
  },
  Object {
    "code": "KY",
    "name": "CAYMAN ISLANDS",
  },
  Object {
    "code": "KZ",
    "name": "KAZAKHSTAN",
  },
  Object {
    "code": "LA",
    "name": "LAO (PEOPLE'S DEM. REP.)",
  },
  Object {
    "code": "LB",
    "name": "LEBANON",
  },
  Object {
    "code": "LC",
    "name": "SAINT LUCIA",
  },
  Object {
    "code": "LI",
    "name": "LIECHTENSTEIN",
  },
  Object {
    "code": "LK",
    "name": "SRI LANKA",
  },
  Object {
    "code": "LR",
    "name": "LIBERIA",
  },
  Object {
    "code": "LS",
    "name": "LESOTHO",
  },
  Object {
    "code": "LT",
    "name": "LITHUANIA",
  },
  Object {
    "code": "LU",
    "name": "LUXEMBOURG",
  },
  Object {
    "code": "LV",
    "name": "LATVIA",
  },
  Object {
    "code": "LY",
    "name": "LIBYA",
  },
  Object {
    "code": "MA",
    "name": "MOROCCO",
  },
  Object {
    "code": "MC",
    "name": "MONACO",
  },
  Object {
    "code": "MD",
    "name": "Rep. MOLDOVA",
  },
  Object {
    "code": "ME",
    "name": "MONTENEGRO",
  },
  Object {
    "code": "MF",
    "name": "SAINT MARTIN",
  },
  Object {
    "code": "MG",
    "name": "MADAGASCAR",
  },
  Object {
    "code": "MH",
    "name": "MARSHALL ISLANDS",
  },
  Object {
    "code": "MK",
    "name": "MACEDONIA",
  },
  Object {
    "code": "ML",
    "name": "MALI",
  },
  Object {
    "code": "MM",
    "name": "MYANMAR",
  },
  Object {
    "code": "MN",
    "name": "MONGOLIA",
  },
  Object {
    "code": "MO",
    "name": "MACAO",
  },
  Object {
    "code": "MP",
    "name": "NORTHERN MARIANA ISLANDS",
  },
  Object {
    "code": "MQ",
    "name": "MARTINIQUE",
  },
  Object {
    "code": "MR",
    "name": "MAURITANIA",
  },
  Object {
    "code": "MS",
    "name": "MONTSERRAT",
  },
  Object {
    "code": "MT",
    "name": "MALTA",
  },
  Object {
    "code": "MU",
    "name": "MAURITIUS",
  },
  Object {
    "code": "MV",
    "name": "MALDIVES",
  },
  Object {
    "code": "MW",
    "name": "MALAWI",
  },
  Object {
    "code": "MX",
    "name": "MEXICO",
  },
  Object {
    "code": "MY",
    "name": "MALAYSIA",
  },
  Object {
    "code": "MZ",
    "name": "MOZAMBIQUE",
  },
  Object {
    "code": "NA",
    "name": "NAMIBIA",
  },
  Object {
    "code": "NC",
    "name": "NEW CALEDONIA",
  },
  Object {
    "code": "NE",
    "name": "NIGER",
  },
  Object {
    "code": "NF",
    "name": "NORFOLK ISLAND",
  },
  Object {
    "code": "NG",
    "name": "NIGERIA",
  },
  Object {
    "code": "NI",
    "name": "NICARAGUA",
  },
  Object {
    "code": "NL",
    "name": "NETHERLANDS",
  },
  Object {
    "code": "NO",
    "name": "NORWAY",
  },
  Object {
    "code": "NP",
    "name": "NEPAL",
  },
  Object {
    "code": "NR",
    "name": "NAURU CENTRAL PACIFIC",
  },
  Object {
    "code": "NU",
    "name": "NIUE",
  },
  Object {
    "code": "NZ",
    "name": "NEW ZEALAND",
  },
  Object {
    "code": "OM",
    "name": "OMAN",
  },
  Object {
    "code": "PA",
    "name": "PANAMA (REP.)",
  },
  Object {
    "code": "PE",
    "name": "PERU",
  },
  Object {
    "code": "PF",
    "name": "FRENCH POLYNESIA",
  },
  Object {
    "code": "PG",
    "name": "PAPUA NEW GUINEA",
  },
  Object {
    "code": "PH",
    "name": "PHILIPPINES",
  },
  Object {
    "code": "PK",
    "name": "PAKISTAN",
  },
  Object {
    "code": "PL",
    "name": "POLAND",
  },
  Object {
    "code": "PM",
    "name": "ST. PIERRE AND MIQUELON",
  },
  Object {
    "code": "PN",
    "name": "PITCAIRN",
  },
  Object {
    "code": "PR",
    "name": "PUERTO RICO",
  },
  Object {
    "code": "PS",
    "name": "PALESTINIAN TERRITORY",
  },
  Object {
    "code": "PT",
    "name": "PORTUGAL",
  },
  Object {
    "code": "PW",
    "name": "PALAU",
  },
  Object {
    "code": "PY",
    "name": "PARAGUAY",
  },
  Object {
    "code": "QA",
    "name": "QATAR",
  },
  Object {
    "code": "RE",
    "name": "REUNION",
  },
  Object {
    "code": "RO",
    "name": "ROMANIA",
  },
  Object {
    "code": "RS",
    "name": "REPUBLIC OF SERBIA",
  },
  Object {
    "code": "RU",
    "name": "RUSSIAN FEDERATION",
  },
  Object {
    "code": "RW",
    "name": "RWANDA",
  },
  Object {
    "code": "SA",
    "name": "SAUDI ARABIA",
  },
  Object {
    "code": "SB",
    "name": "SOLOMON ISLANDS",
  },
  Object {
    "code": "SC",
    "name": "SEYCHELLES",
  },
  Object {
    "code": "SD",
    "name": "SUDAN",
  },
  Object {
    "code": "SE",
    "name": "SWEDEN",
  },
  Object {
    "code": "SG",
    "name": "REP. OF SINGAPORE",
  },
  Object {
    "code": "SH",
    "name": "SAINT HELENA",
  },
  Object {
    "code": "SI",
    "name": "SLOVENIA",
  },
  Object {
    "code": "SJ",
    "name": "SVALBARD AND JAN MAYEN ISLANDS",
  },
  Object {
    "code": "SK",
    "name": "SLOVAKIA",
  },
  Object {
    "code": "SL",
    "name": "SIERRA LEONE",
  },
  Object {
    "code": "SM",
    "name": "SAN MARINO",
  },
  Object {
    "code": "SN",
    "name": "SENEGAL",
  },
  Object {
    "code": "SO",
    "name": "SOMALIA",
  },
  Object {
    "code": "SR",
    "name": "SURINAME",
  },
  Object {
    "code": "SS",
    "name": "SOUTH SUDAN",
  },
  Object {
    "code": "ST",
    "name": "SAO TOME AND PRINCIPE",
  },
  Object {
    "code": "SV",
    "name": "EL SALVADOR",
  },
  Object {
    "code": "SX",
    "name": "SINT MAARTEN",
  },
  Object {
    "code": "SY",
    "name": "SYRIA",
  },
  Object {
    "code": "SZ",
    "name": "SWAZILAND",
  },
  Object {
    "code": "TA",
    "name": "TRISTAN DA CUNHA",
  },
  Object {
    "code": "TC",
    "name": "TURKS AND CAICOS ISLANDS",
  },
  Object {
    "code": "TD",
    "name": "CHAD",
  },
  Object {
    "code": "TF",
    "name": "FRENCH SOUTHERN TERRITORIES",
  },
  Object {
    "code": "TG",
    "name": "TOGO",
  },
  Object {
    "code": "TH",
    "name": "THAILAND",
  },
  Object {
    "code": "TJ",
    "name": "TAJIKISTAN",
  },
  Object {
    "code": "TK",
    "name": "TOKELAU",
  },
  Object {
    "code": "TL",
    "name": "TIMOR-LESTE",
  },
  Object {
    "code": "TM",
    "name": "TURKMENISTAN",
  },
  Object {
    "code": "TN",
    "name": "TUNISIA",
  },
  Object {
    "code": "TO",
    "name": "TONGA",
  },
  Object {
    "code": "TR",
    "name": "TURKEY",
  },
  Object {
    "code": "TT",
    "name": "TRINIDAD AND TOBAGO",
  },
  Object {
    "code": "TV",
    "name": "TUVALU",
  },
  Object {
    "code": "TW",
    "name": "TAIWAN",
  },
  Object {
    "code": "TZ",
    "name": "TANZANIA (UNITED REP.)",
  },
  Object {
    "code": "UA",
    "name": "UKRAINE",
  },
  Object {
    "code": "UG",
    "name": "UGANDA",
  },
  Object {
    "code": "UM",
    "name": "UNITED STATES MINOR OUTLYING ISLANDS",
  },
  Object {
    "code": "US",
    "name": "UNITED STATES",
  },
  Object {
    "code": "UY",
    "name": "URUGUAY",
  },
  Object {
    "code": "UZ",
    "name": "UZBEKISTAN",
  },
  Object {
    "code": "VA",
    "name": "VATICAN",
  },
  Object {
    "code": "VC",
    "name": "SAINT VINCENT AND THE GRENADINES (ANTILLES)",
  },
  Object {
    "code": "VE",
    "name": "VENEZUELA",
  },
  Object {
    "code": "VG",
    "name": "VIRGIN ISLANDS (BRITISH)",
  },
  Object {
    "code": "VI",
    "name": "VIRGIN ISLANDS (U.S.)",
  },
  Object {
    "code": "VN",
    "name": "VIET NAM",
  },
  Object {
    "code": "VU",
    "name": "VANUATU",
  },
  Object {
    "code": "WF",
    "name": "WALLIS AND FUTUNA ISLANDS",
  },
  Object {
    "code": "WS",
    "name": "SAMOA",
  },
  Object {
    "code": "XK",
    "name": "KOSOVO",
  },
  Object {
    "code": "YE",
    "name": "YEMEN",
  },
  Object {
    "code": "YT",
    "name": "MAYOTTE",
  },
  Object {
    "code": "ZA",
    "name": "SOUTH AFRICA",
  },
  Object {
    "code": "ZM",
    "name": "ZAMBIA",
  },
  Object {
    "code": "ZW",
    "name": "ZIMBABWE",
  },
  Object {
    "code": undefined,
    "name": undefined,
  },
]
`;
